src/query/ast/tests/it/testdata/statement.txt

```diff
@@ -3240,6 +3240,114 @@ AlterView(
 )
 
 
+---------- Input ----------
+show views
+---------- Output ---------
+SHOW VIEWS
```

Comment:
Need sqllogic test about show view / desc view like this: 06_0004_show_tables.test .

And also need stateless about privilege test in 18_rbac. like this : 18_0003_db_visibility.sh